admin/app/views/commercial/liveBlogTopSponsorships.scala.html

```diff
@@ -20,16 +20,20 @@ <h1>LiveBlog Top Sponsorships</h1>
 
     <p>Last updated: @if(report.updatedTimeStamp) { @{report.updatedTimeStamp} } else { never }</p>
 
-    <p>Pages will show a liveblog top slot if you set up a LineItem in GAM with the following parameters:</p>
+    <p>Pages will show a live blog top slot if you set up a line item in GAM with the following parameters:</p>
     <ol>
         <li>Is a Sponsorship</li>
         <li>Targets the <pre>liveblog-top</pre> slot</li>
         <li>Targets the <pre>culture</pre>,<pre>sport</pre> or <pre>football</pre> section</li>
         <li>Targets the <pre>liveblog</pre> content type</li>
         <li>Targets the <pre>mobile</pre> breakpoint</li>
-        <li>Targets an edition</li>
+        <li>[Optional] Targets an edition</li>
+        <li>[Optional] Targets a keyword</li>
     </ol>
 
+    <strong>ANY OTHER TARGETING WILL CAUSE THE SLOT TO APPEAR UNINTENTIONALLY</strong>
```

Comment:
I really like that you added this in here so we don't have to encounter this issue again in the future!